pkg/host-disk/host-disk.go

```diff
@@ -99,11 +99,6 @@ func ReplacePVCByHostDisk(vmi *v1.VirtualMachineInstance, clientset kubecli.Kube
 			}
 			// PersistenVolumeClaim is replaced by HostDisk
 			volumeSource.PersistentVolumeClaim = nil
-			// Set ownership of the disk.img to qemu
-			if err := ephemeraldiskutils.DefaultOwnershipManager.SetFileOwnership(file); err != nil && !os.IsNotExist(err) {
-				log.Log.Reason(err).Errorf("Couldn't set Ownership on %s: %v", file, err)
-				return err
-			}
```

Comment:
if this is failing, then virt-launcher must be trying to set the permissions as well in order for this not to cause any issues. with non-root virt-launcher, we can't guarantee that the permissions will get set, so virt-handler actually needs to be performing this action.

rather than remove this, i'd suggest fixing it. Otherwise non root won't work unless the disk on the PVCs filesystem has the correct ownership to begin with.

Reply:
This should always fail as the path is from the launcher view and not from the handler. Therefore this just removes code that doesn't work. I am working on follow-up to unify this handling with nonroot and DVs. 

Non-root is not hit by this because it has https://github.com/kubevirt/kubevirt/blob/main/pkg/virt-handler/non-root.go#L77 b

Reply:
what I'm saying is that the bug is that the file ownership is not set by virt-handler like we'd expect. I don't think removing the non-executing code is a fix, or really useful. Instead we need this to actually work.

If you're working on that solution in another pr, then remove this non-working logic there when it's replaced by something that actually does work.